pallet/futurepass/src/tests.rs

```diff
@@ -15,17 +15,19 @@
 
 use super::*;
 use crate::mock::*;
-use frame_support::traits::tokens::{fungibles::Mutate, Preservation};
+use frame_support::traits::tokens::{fungibles::Mutate, Fortitude, Preservation};
 use hex_literal::hex;
 use seed_pallet_common::test_prelude::*;
 use seed_runtime::{impls::ProxyType, Inspect};
 use sp_runtime::traits::Hash;
 
 type MockCall = crate::mock::RuntimeCall;
 
-// ProxyDepositBase + ProxyDepositFactor * 1(num of delegates) + 1 for Existential deposit
+// ProxyDepositBase + ProxyDepositFactor * 1(num of delegates)
+// + 1 for Existential deposit of funder
 const FP_CREATION_RESERVE: Balance = 148 + 126 + 1;
-const FP_DELEGATE_RESERVE: Balance = 126 * 1 + 1; // ProxyDepositFactor * 1(num of delegates)
+// ProxyDepositFactor * 1(num of delegates) + Existential Deposit
+const FP_DELEGATE_RESERVE: Balance = 126 * 1 + 1;
```

Comment:
```suggestion
const FP_DELEGATE_RESERVE: Balance = 126 * 1 ;
```
Delegate reserve should not include the ED. By the time the delegates are added, FP should have been created already.

Reply:
Might need to review again how we redirect balance calls via AssetExt. you can leave this to me.

Reply:
[da66f0a](https://github.com/futureversecom/trn-seed/pull/844/commits/da66f0a426e5952863ae26abca7475ffd19fec8c)